database/index.js

```diff
@@ -51,16 +51,28 @@ const insertAvailability = (availability, count, callback) => {
 
 const updateAvailability = (availability, count, callback) => {
   console.log(availability);
-  connection.query('INSERT INTO availability (availability_date, price, is_available, listings_id) VALUES (?, ?, ?, ?)', [availability.date, availability.price, availability.is_available, availability.listings_id], function (err, results, fields) {
+  connection.query(`SELECT * from availability INNER JOIN listings ON listings.id = ${availability.listings_id} where availability.availability_date = '${availability.date}' ORDER BY availability.id DESC LIMIT 1`, (err, listingsResults, fields) => {
     if (err) {
       callback(err, null);
     } else {
-      console.log(availability.listings_id);
-      callback(null, results);
+      connection.query('INSERT INTO availability (availability_date, price, is_available, listings_id) VALUES (?, ?, ?, ?)', [availability.date, `${listingsResults[0].price}`, availability.is_available, availability.listings_id], (err, availabilityResults, fields) => {
```

Comment:
No need to declare availabilityResults and fields if they are not going to be used. (Did the linter not complain?)
Similar comment for line 62 for neighborhoodResults.

Reply:
@christine-w availabilityResults and destinationsResults are part of the key queries and will incorporate them when I'm ready to publish them to the message bus